packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -860,7 +861,7 @@ function Panel({
             sx={sx}
             className={clsx(className, classes.FilteredActionList)}
             // needed to explicitly enable announcements for deprecated FilteredActionList, we can remove when we fully remove the deprecated version
-            announcementsEnabled
+            announcementsEnabled={usingRemoveActiveDescendant}
```

Comment:
this change doesn't look right, what happened to the deprecated actionlist consideration?

Reply:
So basically this is part of the fix to prevent double announcements